src/SFA.DAS.Activities/ActivityType.cs

```diff
@@ -34,6 +34,7 @@ public enum ActivityType
 
         [Category(ActivityTypeCategory.AccountAdmin)]
         [Localizer(typeof(LegalEntityAddedActivityLocalizer))]
+        [Action("agreements", "EmployerAgreement")]
```

Comment:
I don't think an `agreements` action exists?

Reply:
that's me looking at the wrong thing lol I have no idea why I picked the URL mapping value. Changed.